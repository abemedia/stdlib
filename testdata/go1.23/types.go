package test

import (
	"golang.org/x/exp/constraints" // want "The golang.org/x/exp/constraints package import is no longer necessary"
)

type _ interface {
	constraints.Ordered // want "golang.org/x/exp/constraints.Ordered can be replaced with cmp.Ordered"
}

func _[T constraints.Ordered](v T) {} // want "golang.org/x/exp/constraints.Ordered can be replaced with cmp.Ordered"
