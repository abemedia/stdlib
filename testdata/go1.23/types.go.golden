package test

import (
	// want "The golang.org/x/exp/constraints package import is no longer necessary"

	"cmp"
)

type _ interface {
	cmp.Ordered // want "golang.org/x/exp/constraints.Ordered can be replaced with cmp.Ordered"
}

func _[T cmp.Ordered](v T) {} // want "golang.org/x/exp/constraints.Ordered can be replaced with cmp.Ordered"
